src/murmulator/ps2kbd_mrmltr.h

```diff
@@ -0,0 +1,61 @@
+// You may use, distribute and modify this code under the
```

Comment:
This is similar to code in pico-emu-utils ps2kbd.cpp, ps2kbd.h & ps2kbd.pio
https://github.com/fruit-bat/pico-emu-utils
Please reuse if possible.